### PR TITLE
Fix changes in SSD for backward compatibility

### PR DESCRIPTION
**Pull request by Genuster**
@larsoner this should do it for #13326

### CODE REVIEW

**Comment by larsoner** [comment]
Can you add a tiny test that at least calls the method in `test_ssd.py`, and then also make any trivial change to [examples/decoding/ssd_spatial_filters.py](https://github.com/mne-tools/mne-python/pull/13326/files#diff-34756a2ff9387fcb986524f8113528152f0300d5a3950d384803d807eb3a9724) to get CircleCI to run the example? Then we can merge. Thanks for the quick fix!

**Comment by Genuster** [comment]
If I add the method call in tests then I can remove the vulture entry I've just added?

**Comment by larsoner** [comment]
Yep! Also I'll add a label that this doesn't need a change log entry

File: mne/decoding/ssd.py [hunk 1]
```diff
@@ -289,6 +289,39 @@ def fit_transform(self, X, y=None, **fit_params):
...
+        spec_ratio : array, shape (n_channels)
+            Array with the sprectal ratio value for each component.
+        sorter_spec : array, shape (n_channels)
+            Array of indices for sorting spec_ratio.
+
+        References
+        ----------
+        .. footbibliography::
+        """
+        from ._mod_ged import _get_spectral_ratio
```
Comment (larsoner) [inline mne/decoding/ssd.py]: There is already a `from ._mod_ged` above so this should move up there. We only really want to nest imports that give circular import problems

**Comment by larsoner** [comment]
Thanks for the quick fix @Genuster !
